pkg/controller/actions/gc/action_gc.go

```diff
@@ -128,7 +136,9 @@ func NewAction(opts ...ActionOpts) actions.Fn {
 		opt(&action)
 	}
 
-	action.selector = labels.SelectorFromSet(action.labels)
+	if len(action.labels) > 0 {
```

Comment:
so this is saying
if the caller (component) set `gc.WithLabel(...)` , use it to convert to selector
if not, selector is using  "components.opendatahub.io/part-of:componentsv1.XXXName"

Reply:
correct